File: frontend/src/app/utility/employee.ts
export interface Employee{
    
    id:number;
    name:string;
    department:string;
    status:string;
    phoneno:number;
    country:{
        cid:number,
        cname:string
    }
    createddtm:number;
    createdby:string;
    updateddtm:number;
    updatedby:string|null;  
  
  }

File: backend/src/main/java/com/pm/EmployeeManagementPortal/entity/Employee.java
package com.pm.EmployeeManagementPortal.entity;

import java.util.Date;

import jakarta.persistence.Entity;
import jakarta.persistence.FetchType;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Entity
@Getter
@Setter
@ToString
@AllArgsConstructor
@NoArgsConstructor
public class Employee {
	
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	int id;
	
	String name;
	String department;
	String status;
	String phoneno;
	double salary;
	Date createddtm;
	String createdby;
	Date updateddtm;
	String updatedby;
	String emailid;
	
	@ManyToOne(fetch = FetchType.EAGER)
	@JoinColumn(name = "cid")
	Country country;
	
	
	
}
